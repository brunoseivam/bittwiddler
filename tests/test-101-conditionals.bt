main {
    var x : int16 = 0;
    var y : int32 = if x == 0 {
        emit("x is 0\n");
        1;
    } else {
        emit("x is not 0\n");
        2;
    };
    emit("y={y}\n");
}
