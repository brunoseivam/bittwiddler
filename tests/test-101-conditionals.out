x is 0
y=1
